indico/modules/events/abstracts/__init__.py

```diff
@@ -118,3 +118,6 @@ def _is_visible(conf):
                         position=9, visible=lambda event: event.has_feature('abstracts'), static_site=True)
     yield MenuEntryData(title=_("My Abstracts"), name='user_abstracts', visible=_is_visible,
                         endpoint='abstracts.my_abstracts', position=0, parent='call_for_abstracts')
+    yield MenuEntryData(title=_("Submit Abstract"), name='abstract_submission',
+                        visible=lambda event: event.has_feature('abstracts'),
```

Comment:
We could have a message "The call for abstracts is closed. Submission period: {date1} to {date2}". This way we don't have to check whether we have to put "yet" or anything else, it covers all possible cases.